addons/bestja_organization/views/organization.xml

```diff
@@ -132,6 +132,7 @@
             <field name="model">organization</field>
             <field name="arch" type="xml">
                 <search>
+                    <field name="name"/>
```

Comment:
Is it an intended change? It doesn’t seem to be related to the rest of the commit.